ci-operator/step-registry/quay/deploy-gcp-gcs/quay-deploy-gcp-gcs-commands.sh

```
@@ -0,0 +1,437 @@
+    managed: true
+EOF
+
+echo "Waiting for Quay to become ready (timeout: 15m)..." >&2
+for i in $(seq 1 90); do
+  status="$(oc -n "${QUAY_NS}" get quayregistry quay -o jsonpath='{.status.conditions[?(@.type=="Available")].status}' 2>/dev/null || true)"
+  if [[ "$status" == "True" ]]; then
+    echo "Quay is ready (after $((i * 10))s)" >&2
+    oc -n "${QUAY_NS}" get quayregistries -o yaml >"$ARTIFACT_DIR/quayregistries.yaml"
+    oc get quayregistry quay -n "${QUAY_NS}" -o jsonpath='{.status.registryEndpoint}' > "$SHARED_DIR"/quayroute || true
+    quay_route=$(oc get quayregistry quay -n "${QUAY_NS}" -o jsonpath='{.status.registryEndpoint}') || true
+    curl -k -X POST $quay_route/api/v1/user/initialize --header 'Content-Type: application/json' \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)